index.html

```diff
@@ -12,10 +12,76 @@
       align-items: center;
       flex-direction: column;
     }
+
+    .hidden {
+      display: none;
+    }
+
+    .inline-text {
+      display: inline-block;
+      position: fixed;
+      top: 0;
```

Comment:
```suggestion
      top: 1em;
      right: 1em;
```

Reply:
Not sure about `margin-right` vs. `right`.

Reply:
Changed to only use right and converted p to div to avoid browser default top margin